src/main/java/org/prebid/server/bidder/model/BidderErrorInfo.java

```diff
@@ -0,0 +1,14 @@
+package org.prebid.server.bidder.model;
+
+import lombok.Builder;
+import lombok.Value;
+
+/**
+ * Stores error information: error code and message
+ */
+@Builder
+@Value
+public class BidderErrorInfo {
```

Comment:
This class has to be renamed to `ExtBidderError` and moved to `org.prebid.server.proto.openrtb.ext.response` as it really belongs there (it is a part of `proto`col, in particular it represents OpenRTB response)